### DISCUSSION TITLE
Do I need this if I already use Nextcloud Client to sync files?

### DISCUSSION THREAD

**Discussion by ghost** [remotely-save/remotely-save#861] [Q&A]
Hi,
As the title says, I'm wondering if I need it to sync a vault if I already use Nextcloud Client?
I previously used AppFlowy which creates zillions of files that are saved every picosecond. This implies zillions of notifications (which I had to deactivate just because of AppFlowy).
I think I understand that with this module, I can have my vault anywhere on my local storage and have it synced with my Nextcloud instance (but at the root level?). I don't really care about where my vaults are on my local disk.
Does it bring other features I may have missed or can I simply go with Nextcloud Client?

**Comment by fyears** [comment]
for example if you are using obsidian ios, you might be unable to use nextcloud app together.
